lib/common/options.js

```diff
@@ -60,6 +60,25 @@ Options.from = function (_options) {
     : options.rootDir
   options.buildDir = path.resolve(options.rootDir, options.buildDir)
 
+  // Dummy proof buildDir and generate.dir as those folders
+  //  will be completely removed when used
+  if (options.rootDir.startsWith(options.buildDir)) {
```

Comment:
Not sure. But for more security maybe we have to `path.resolve(options.rootDir)`

Reply:
Not needed :D Sorry

Reply:
Already did on line 44 :)

Reply:
Saw it too late :)